model/pipeline/src/main/proto/metrics.proto

```diff
@@ -401,6 +402,17 @@ message MonitoringInfo {
   // as Stackdriver will be able to aggregate the metrics using a subset of the
   // provided labels
   map<string, string> labels = 4;
+
+  // This indicates when the first value was populated by the SDK Harness.
```

Comment:
Maybe "This indicates the start of the time range over which this value was measured"?